### PR TITLE
feat(docs): FRMW-165 - Migrate rxBulkSelect docs to ngdocs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-168

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxBulkSelect/README.md [hunk 1]
```diff
@@ -2,65 +2,6 @@
...
-
-rxBulkSelect is an **attribute** directive that decorates a `<table>` to provide an api for the rest of the directives.  It requires the list that is used to populate the table and a unique key (e.g. anything that doesn't interfere with the object for a row) to save the selection state of each row.  These are the `bulk-source` and `selected-key` attributes, respectively.  The optional `resource-name` attribute is used to write the message that is displayed when rows are selected, e.g. "2 servers are selected".  When not included, it defaults to the `bulk-source` variable name (with the trailing "s" removed, if applicable).
-
-```html
-<table rx-bulk-select bulk-source="servers" selectedKey="rowIsSelected"></table>
-```
-
-The directive is also responsible for adding a row to the table header that indicates how many rows are selected and contains buttons to select or deselect all the rows at once.
-
-Each of the following directives are only valid when used inside a table with rxBulkSelect.
```
Comment (CITguy) [inline src/rxBulkSelect/README.md]: Please combine with documentation for `rxBulkSelect` [directive](https://github.com/rackerlabs/encore-ui/blob/master/src/rxBulkSelect/rxBulkSelect.js#L22-L39).

Comment (bry4n) [inline src/rxBulkSelect/README.md]: Done. let me know if I am missing something.

**Comment by 100stacks** [comment]
LGTM

**Comment by CITguy** [comment]
LGTM

**Pull request merged**